pygments/lexers/ruby.py

```diff
@@ -328,9 +328,13 @@ def intp_string_callback(self, match, ctx):
         ],
         'funcname': [
             (r'\(', Punctuation, 'defexpr'),
-            (r'(?:([a-zA-Z_]\w*)(\.))?'
-             r'([a-zA-Z_]\w*[!?]?|\*\*?|[-+]@?|'
-             r'[/%&|^`~]|\[\]=?|<<|>>|<=?>|>=?|===?)',
+            (r'(?:([a-zA-Z_]\w*)(\.))?'  # optional scope name, like "self."
+             '('
+                '[a-zA-Z\u0080-\uffff][a-zA-Z0-9_\u0080-\uffff]*[!?=]?'  # method name
+                '|!=|!~|=~|\\*\\*?|[-+!~]@?|[/%&|^]|<=>|<[<=]?|>[>=]?|===?'  # or operator override
+                r'|\[\]=?'  # or element reference/assignment override
+                '|`'  # or the undocumented backtick override
```

Comment:
Any particular reason for mixing raw and non-raw strings here? I'd prefer to have everything consistent (i.e. `r'...'`) because it makes editing less error prone.

Reply:
I standardized on plain strings for the string block so I could still use the Unicode escapes.

Reply:
That's perfectly fine, but for my curiosity: Aren't Unicode escapes supported by the regex engine nowadays?

Reply:
You're right! Support was added in Python 3.3. I've added a commit so that the block isn't just self-consistent; it's now consistent with the rest of the surrounding raw strings. Thanks for pointing this out! 😃